.github/workflows/ci.yml

```diff
@@ -482,14 +482,23 @@ jobs:
         run: echo OPENSSL_SRC_PERL=C:/Strawberry/perl/bin/perl >> $GITHUB_ENV
         timeout-minutes: 1
 
+      # By default, `libparsec_crypto` uses Rust Crypto
+      - name: Test sodiumoxide
+        if: >-
+          steps.rust-changes.outputs.run == 'true'
+          && startsWith(matrix.os, 'ubuntu-')
+        shell: bash
+        run: |
+          set -ex
+          cargo test --profile ci-rust --package libparsec_crypto --features use-sodiumoxide
+        timeout-minutes: 1
```

Comment:
1 minutes to compile & execute the crypto tests seems very short, increase it to 15 min ?